modules/gdscript/gdscript_editor.cpp

```diff
@@ -1898,7 +1898,7 @@ static bool _guess_identifier_type(GDScriptParser::CompletionContext &p_context,
 	// Look in blocks first.
 	int last_assign_line = -1;
 	const GDScriptParser::ExpressionNode *last_assigned_expression = nullptr;
-	GDScriptParser::DataType id_type;
+	GDScriptParser::DataType id_type, weak_type;
```

Comment:
```suggestion
	GDScriptParser::DataType id_type;
	GDScriptParser::DataType weak_type;
```
Prefer to keep variables on their own line, two ones on one line is very hard to miss and hurts readability